### ISSUE TITLE
onFinish() in IUploader.OnFinishUploaderHandler() is call two times.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#105]
```
What steps will reproduce the problem?
1. Choose a file to upload whom size is bigger than the max size (in my case, 
5Mb)

2. Submit the SingleUploader

3. An error message appear about the size of the file

4. Choose a other file to upload whom size is smaller than the max size

5. Submit the SingleUploader

6. Here, the onFinish() methode of OnFinishUploaderHandler is call two times, 
but my Servlet is call one time.



What is the expected output? What do you see instead?
I expected that the onFinish method is call only one time.



What version of the product are you using? On what operating system?
GWTUpload 0.6.1 with IE8 on Window 7



Please provide any additional information below.
--
```

Original issue reported on code.google.com by `theophil...@gmail.com` on 18 May 2011 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
I test with 0.6.3 version, and in this case, the onFinish() method is called 
three times
```

Original comment by `theophil...@gmail.com` on 18 May 2011 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
Please test with gwtupload-0.6.4-SNAPSHOT it should be fixed here, 0.6.1 and 
0.6.3 are buggy.
```

Original comment by `manuel.carrasco.m` on 5 Jul 2011 at 1:11
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 9 Nov 2011 at 7:02
- Changed state: **Done**
